Tetris_formal.py

```diff
@@ -95,47 +139,34 @@ def get_direction(self, color):
             [color, self.x - 40, self.y - 40],
         ]
         return up, down, left, right
-    
-    
+
     def get_color(self):
-        num = random.randint(1,2)
-        
-        if num == 1 :
-            color = (70,90,120)
-        
-        elif num == 2 :
-            
-            color = pygame.image.load('/Users/coolguy/Documents/picture_for_L.jpg')
-            color = pygame.transform.scale(color,(40,40))
-            color.blit(color,(self.x, self.y))
-            
-       
+        """
+        同上
+        """
+        color = self.color_or_picture(
+            (70, 90, 120), "/Users/coolguy/Documents/picture_for_L.jpg"
+        )
+
         return color
-    
-   
-        
-        
-            
-            
-        
 
 
-class J_block():
+class JBlock(InitializeData):
+    """
+    同上
+    """
 
-    def __init__(self, x, y):
-        self.x = x
-        self.y = y
-    
-    
     def get_direction(self, color):
-        
-        
+        """
```

Comment:
Ditto. The description doesn't mean anything. Please briefly describe the function.